doc/proposal.texi

```diff
@@ -348,7 +348,21 @@ be found. Some othernets do not (regularly) distribute a nodediff file, but
 just hatch a new nodelist every few weeks. In this case, you need the
 @code{FullUpdate} statement.
 
-@node defaultzone, nodelistformat, fullupdate, NLTools
+@node dailynodelist, defaultzone, fullupdate, NLTools
+@subsection dailynodelist
+@table @asis
+@item Syntax:
+@code{Dailynodelist <bool>}
+@item Example:
+@code{Dailynodelist on} or@code{Dailynodelist off}
+@end table
+
+If you set @code{Dailynodelist} to @code{on}, NLTools do a fullupdate of the nodelist on a daily basis, instead to check if the new nodelist is 7 days newer. You have to set fullupdate and dailynodelist statements and not the diffupdate statement for the daily nodelist processing.
```

Comment:
NLTools do a fullupdate of the nodelist on a daily basis, instead to check => NLTools do a full update of the nodelist on a daily basis, instead of checking